tests/unit/utils/test_fs.py

```diff
@@ -39,6 +39,15 @@ def test(self):
         self.assertIs(type(dir_size), str)
         self.assertEqual(dir_size, str(actual_dir_size))
 
+    def test_path_object_and_str_are_valid_types(self):
+        dvcignore = DvcIgnoreFilter(self.root_dir)
+        file_time, file_size = get_mtime_and_size(self.DATA, dvcignore)
+        file_object_time, file_object_size = get_mtime_and_size(
+            PathInfo(self.DATA), dvcignore
+        )
+        self.assertEqual(file_time, file_object_time)
+        self.assertEqual(file_size, file_object_size)
```

Comment:
Not sure this test is valuable: 
- this is tested via functional tests anyway 
- this doesn't test for non-dir argument

I would just drop it.

Reply:
@Suor @efiop How should I make the required changes?

Reply:
We are verifying that it works the same with str and path_infos and for that the test is suitable and valuable. Let's keep it.

Reply:
@algomaster99 But `this doesn't test for non-dir argument` is a valid one(though it is probably meant to say "dir" instead of "non-dir"). Let's also run this test with `self.DATA_DIR`, so we verify that this works for dirs as well.